README.md

```diff
@@ -271,10 +271,11 @@ class Lambda4 {
 
 Remember the formula example from the first section? Interface `Formula` defines a default method `sqrt` which can be accessed from each formula instance including anonymous objects. This does not work with lambda expressions.
 
-Default methods **cannot** be accessed from within lambda expressions. The following code does not compile:
+Default methods **cannot** be accessed from within lambda expressions.
 
 ```java
-Formula formula = (a) -> sqrt(a * 100);
+Formula formula1 = (a) -> sqrt(a * 100); // Not compile
+Formula formula2 = (a) -> Math.sqrt(a * 100); // Compile
```

Comment:
I don't  think `Math.sqrt` example should be added here.
The earlier example was sufficient to show that default methods can't be called within the lambda expressions.
This will confuse the reader.